schema/change.json

```diff
@@ -26,6 +26,14 @@
         },
         "tombstone": {
             "type": "boolean"
+        },
+        "labels": {
```

Comment:
It might be nice to extract the next couple of lines in it's own object and re-use it from `admin-stats-response.json`